File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/event/exception/AlreadyOpenStatusException.java
package band.gosrock.domain.domains.event.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class AlreadyOpenStatusException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new AlreadyOpenStatusException();

    private AlreadyOpenStatusException() {
        super(EventErrorCode.ALREADY_OPEN_STATUS);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/event/exception/AlreadyOpenStatusException.kt
package band.gosrock.domain.domains.event.exception

import band.gosrock.common.exception.DuDoongCodeException

class AlreadyOpenStatusException private constructor() : DuDoongCodeException(EventErrorCode.ALREADY_OPEN_STATUS) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = AlreadyOpenStatusException()
    }
}
